src/main/java/me/zeroeightsix/kami/command/commands/CoordsCommand.java

```diff
@@ -0,0 +1,85 @@
+package me.zeroeightsix.kami.command.commands;
+
+import me.zeroeightsix.kami.command.Command;
+import me.zeroeightsix.kami.command.syntax.ChunkBuilder;
+import me.zeroeightsix.kami.util.Coord;
+
+import java.util.Objects;
+
+import static me.zeroeightsix.kami.util.CoordUtil.coordsLogToArray;
+import static me.zeroeightsix.kami.util.CoordUtil.writePlayerCoords;
+import static me.zeroeightsix.kami.util.MessageSendHelper.sendChatMessage;
+import static me.zeroeightsix.kami.util.MessageSendHelper.sendRawChatMessage;
+
+/**
+ * @author wnuke
+ * Created by wnuke on 17/04/20
+ */
+
+public class CoordsCommand extends Command {
+    public CoordsCommand() {
+        super("pos", new ChunkBuilder()
+                .append("save|list", true)
+                .append("name|searchterm", false)
+                .append("chunk|showDateTime", false)
+                .build());
+        setDescription("Log the current coordinates.");
+    }
+
+    public void call(String[] args) {
+        if (args[0] != null) {
```

Comment:
Uh, this will never be null, I think you meant to do `if (args.length >= 1)`

Reply:
It will be null if there is no argument.